src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/SettingsPropertyValue.cs

```diff
@@ -98,6 +98,7 @@ private object Deserialize()
                     {
                         using (MemoryStream ms = new MemoryStream((byte[])SerializedValue))
                         {
+                            // Issue https://github.com/dotnet/runtime/issues/39295 tracks finding an alternative to BinaryFormatter
```

Comment:
Why isn't suppression needed in this file?

Reply:
It's not compiled against net5.0 so won't see the newly-added `[Obsolete]` attributes.

Reply:
Ah, ok.